packages/dashboard-backend/src/local-run/dexHelper.ts

```diff
@@ -96,10 +103,30 @@ export function addDexProxy(server: FastifyInstance) {
     rewritePrefix: '/auth',
     disableCache: true,
   });
+}
+
+export function addAutoresizeHooks(server: FastifyInstance) {
+  server.addHook('onResponse', (request, reply, done) => {
+    if (
+      (request.url.startsWith('/api/') || request.url.startsWith('/dashboard/api/')) &&
+      request.method === 'GET' &&
+      reply.statusCode == 401
+    ) {
+      if (reply.statusCode === 401) {
+        process.env.CLUSTER_ACCESS_TOKEN = '';
+      }
```

Comment:
This condition is always true:

```suggestion
      process.env.CLUSTER_ACCESS_TOKEN = '';
```

Reply:
Good catch. I have fixed it.